application/views/catalog/partials/footer.php

```diff
@@ -45,7 +45,7 @@ function get_search_page_from_url() {
 	var search_page = get_search_page_from_url();
 	set_advanced_form_page(search_page);
 
-	var search_order = 'alpha';
+	var search_order = $('.js-sort-menu').val() || 'alpha';
```

Comment:
`.js-sort-menu` is the sort order `<select>`. We're already going to some pains to keep it in sync with `search_order`, so I figured it wouldn't be the worst thing if I initialized this variable based on that (if it's available)